src/orion/core/cli/serve.py

```diff
@@ -39,7 +40,8 @@ def main(args):
     """Starts an application server to serve http requests"""
     config = experiment_builder.get_cmd_config(args)
 
-    web_api = WebApi(config)
+    storage = setup_storage()
```

Comment:
In WebApi the storage was created based on `config['config']`. It should be created the same way here before being passed to setup_storage() otherwise we cannot configure what storage to use with a local config file.

Reply:
fixed